static/app/components/modals/inviteMembersModal/inviteStatusMessage.tsx

```diff
@@ -39,12 +43,45 @@ export default function InviteStatusMessage({
     const errorCount = statuses.filter(i => i.error).length;
 
     if (willInvite) {
-      const invites = <strong>{tn('%s invite', '%s invites', sentCount)}</strong>;
+      const invites = (
+        <strong data-test-id="sent-invites">
+          {tn('%s invite', '%s invites', sentCount)}
+        </strong>
+      );
+      const failedInvites = (
+        <strong data-test-id="failed-invites">
+          {tn('%s invite', '%s invites', errorCount)}
+        </strong>
+      );
```

Comment:
Nit: recommend pulling this out as a `InviteCount` component with 2 props `isError` and `count`. Also, we may want to use CSS rather than the `strong` tag -- can just set the font weight instead.